server/models/deviceModel.js

```diff
@@ -0,0 +1,27 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+const House = require('./houseModel');
+const DeviceCategory = require('./deviceCategoryModel');
+
+const Device = db.define('device', {
+  id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true,
+  },
+  name: Sequelize.STRING,
+  isActive: Sequelize.BOOLEAN,
+  hardwareKey: Sequelize.STRING,
+  hardwareType: Sequelize.STRING,
+  usageTimeOptions: Sequelize.INTEGER,
```

Comment:
how do we pass in the array of options? (i.e. 15 minutes, 60 minutes, 1,440 minutes)